packages/app-root/src/app/page.js

```diff
@@ -40,7 +40,7 @@ async function getBlogPosts() {
   try {
     const feeds = await Promise.all(BLOG_FEEDS.map(fetchBlogFeed))
     feeds.forEach(feed => {
-      posts = posts.concat(feed.slice(0, 3))
+      posts = posts.concat(feed.slice(0, 4))
```

Comment:
[medium suggestion] This number 4's floating around a bit willy-nilly, it might be worth moving it ~into a config~ into a constant like NUMBAH_OF_BLOG_POSTS = 4 to match BLOG_FEEDS